src/frontend/context/BrevModulContext.ts

```diff
@@ -96,9 +97,18 @@ const brevmalKanVelgesForBehandling = (brevmal: Brevmal, åpenBehandling: IBehan
             );
         case Brevmal.INNHENTE_OPPLYSNINGER_INSTITUSJON:
             return åpenBehandling.årsak === BehandlingÅrsak.SØKNAD;
+        case Brevmal.VARSEL_OM_ÅRLIG_REVURDERING_EØS:
+        case Brevmal.VARSEL_OM_ÅRLIG_REVURDERING_EØS_MED_INNHENTING_AV_OPPLYSNINGER:
+            return åpenBehandling.årsak === BehandlingÅrsak.ÅRLIG_KONTROLL;
     }
 };
 
+const utledFeilmeldingDokumenter = (brevmal: Brevmal) => {
+    return brevmal === Brevmal.VARSEL_OM_ÅRLIG_REVURDERING_EØS_MED_INNHENTING_AV_OPPLYSNINGER
+        ? 'Brevmalen krever at du enten velger dokumenter fra listen over, eller legger til et kulepunkt med fritekst'
+        : `Du må velge minst ett dokument`;
+};
```

Comment:
Gjelder ikke egentlig den første teksten her uansett brevmal? For utledFeilmeldingDokumenter-funksjonen blir bare kalt hvis det er valgt null dokumenter og ingen fritekst. Kanskje ta en avsjekk med Gunn om den teksten kan bli brukt i alle tilfellene

Reply:
Eller vent, fritekst er kanskje ikke synlig for alle brevmalene

Reply:
Men for alle brevmalene som kan velge både fritekst og dokumenter så gir det mening å ha denne feilmeldingen

Reply:
Har sendt spørsmålet vidare til Gunn